internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -166,6 +171,23 @@ static_resources:
                 address: {{ $sink.Address }}
                 port_value: {{ $sink.Port }}
   {{- end }}
+  - connect_timeout: 10s
+    lb_policy: ROUND_ROBIN
+    load_assignment:
+      cluster_name: local_cluster
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 10080
```

Comment:
need update open ports in doc?

Reply:
This doesn't actually open a port it's just a reference to a fake lb_endpoint to satisfy the internal envoy logic of at least 1 endpoint. I used 10080 from the default port 80 listener but can change this in a followup to be the readiness probe or another dummy port if it's confusing.

Reply:
can you address this somewhere, this would be enough.

Reply:
Updated